src/circuits/commitments/commitments.hpp

```diff
@@ -23,6 +23,7 @@ class COMM_gadget : libsnark::gadget<FieldT>
 public:
     COMM_gadget(
         libsnark::protoboard<FieldT> &pb,
+        const libsnark::pb_variable<FieldT> ZERO,
```

Comment:
Same question as above, why not passing ZERO by reference?

Reply:
done